### DISCUSSION TITLE
Eta plugin template directory issue

### DISCUSSION THREAD

**Discussion by doujinreviewers** [lumeland/lume#454] [Q&A] [answered]
Eta v3 can only specify single template directory.
https://github.com/eta-dev/eta/issues/238

And my understanding is that lume's Eta plugin is overriding the only option specification and there is no way for the user to specify `views`.
https://github.com/lumeland/lume/blob/master/plugins/eta.ts#L102

I want to specify the template directory. Is it possible?

**Comment by oscarotero** [comment, marked as answer]
you can configure the plugin to use a diferent path. For example:

```js
site.use(eta({
  includes: "./eta-templates"
}));
```

**Reply by doujinreviewers** [reply]
It went well. Thank you!
